nx584/controller.py

```diff
@@ -346,6 +346,13 @@ def _get_user(self, number):
         return self.users[number]
 
     def process_msg_3(self, frame):
+        try:
+            zone_name_update = self._config.getboolean('config', 'zone_name_update')
```

Comment:
It would be better to do this during init so we're not constantly looking this up in the config file every time we get this frame, nit not a huge deal.